package flume_metrics;

message HadoopStatus {
  message JobStatus {
    required string job_id = 1;
    required float map = 2;
    required float reduce = 3;
  }

  required string timestamp = 1;
  repeated JobStatus job_status = 2;
}
